<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageWidget</class>
 <widget class="QWidget" name="ManageWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>377</width>
    <height>177</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout" stretch="1,2,1">
       <item>
        <widget class="QLabel" name="labelVersion">
         <property name="text">
          <string>最新版本 1.2.0</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLineEdit" name="lineFilepath"/>
         </item>
         <item>
          <widget class="QPushButton" name="btnSelect">
           <property name="text">
            <string>下载游戏</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="labelInfo">
         <property name="text">
          <string>所需空间 23.02GB 可用空间 34.2GB</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QPushButton" name="btnStart">
         <property name="text">
          <string>开始游戏</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
